panic while running ncdumpzone

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4459b07]

goroutine 1 [running]:
github.com/namecoin/ncbtcjson.NewNameScanCmd(...)
	/Users/darren/Sources/go/pkg/mod/github.com/namecoin/ncbtcjson@v0.0.2/namecmds.go:86
github.com/namecoin/ncrpcclient.(*Client).NameScanAsync(0xc0000a20f0, 0x45c0801, 0x2, 0x3e8, 0x8)
	/Users/darren/Sources/go/pkg/mod/github.com/namecoin/ncrpcclient@v0.0.1/names.go:106 +0x67
github.com/namecoin/ncrpcclient.(*Client).NameScan(0xc0000a20f0, 0x45c0801, 0x2, 0xc0000003e8, 0xc0000a20f0, 0xc00021fe88, 0x4459112, 0xc00021fe88, 0x400f368)
	/Users/darren/Sources/go/pkg/mod/github.com/namecoin/ncrpcclient@v0.0.1/names.go:112 +0x4b
github.com/namecoin/ncdns/ncdumpzone.Dump(0xc0000a20f8, 0x4799620, 0xc0000a2008, 0x45c2097, 0x8, 0x4799260, 0xc000155aa0)
	/Users/darren/Sources/go/src/github.com/namecoin/ncdns/ncdumpzone/ncdumpzone.go:97 +0x279
main.main()
	/Users/darren/Sources/go/src/github.com/namecoin/ncdns/ncdumpzone/ncdumpzone/ncdumpzone.go:63 +0x35b
```

--- a/namecmds.go
+++ b/namecmds.go
@@ -46,7 +46,7 @@ func NewNameShowCmd(name string, options *NameShowOptions) *NameShowCmd {
 	// Don't send the Options parameter if it's empty.  This saves some
 	// space on the wire, and also ensures compatibility with old Namecoin
 	// Core versions that don't support the Options parameter.
-	if *options == (NameShowOptions{}) {
+	if options != nil && *options == (NameShowOptions{}) {
 		options = nil
 	}
 

--- a/namecmds.go
+++ b/namecmds.go
@@ -83,7 +83,7 @@ func NewNameScanCmd(start string, count *uint32, options *NameScanOptions) *Name
 	// Don't send the Options parameter if it's empty.  This saves some
 	// space on the wire, and also ensures compatibility with old Namecoin
 	// Core versions that don't support the Options parameter.
-	if *options == (NameScanOptions{}) {
+	if options != nil && *options == (NameScanOptions{}) {
 		options = nil
 	}
 

